locales/vi-VN.json

```
@@ -640,6 +640,7 @@
       "count": "{0} bình chọn|{0} bình chọn|{0} bình chọn",
       "ends": "đóng {0}",
       "finished": "kết thúc {0}"
```

Thanks for the update! 🙏🏻 

An additional `,` is necessary at the end of this line:

```suggestion
      "finished": "kết thúc {0}",
```